core/src/main/java/org/apache/accumulo/core/util/UtilWaitThread.java

```diff
@@ -18,17 +18,45 @@
  */
 package org.apache.accumulo.core.util;
 
+import static java.util.concurrent.TimeUnit.MICROSECONDS;
+
+import java.util.concurrent.TimeUnit;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class UtilWaitThread {
   private static final Logger log = LoggerFactory.getLogger(UtilWaitThread.class);
 
+  /**
+   * Sleep for specified duration in milliseconds. If the sleep is interrupted, a message is logged
+   * and the interrupt flag is reset, but otherwise the interrupt is ignored.
+   * <p>
+   *
+   * @deprecated Use {@link UtilWaitThread#sleep(long, TimeUnit)} instead.
+   */
+  @Deprecated(since = "3.0.0")
   public static void sleep(long millis) {
+    sleep(millis, MICROSECONDS);
```

Comment:
Should the time unit here be MILLISECONDS?

Reply:
Resolved with 5f2c1cb00aed8d8